python/reference-apps/python-topic-producer/main.py

```diff
@@ -4,4 +4,4 @@
 }
 
 def run(context, input):
-    return input.map(lambda s: f'producer got: {s}').each(print)
+    return input.map(lambda s: f'producer got: {s}' + "\n").each(print)
```

Comment:
After we no longer close topics, we need this newline character. It's related to a bug in Python runner: https://github.com/scramjetorg/transform-hub/issues/376

Reply:
Perhaps it would be better to add this to the last line in the actual input stream in runner?